acceptance/setup/package/pre-suite/020_install.rb

```diff
@@ -1,9 +1,39 @@
 # frozen_string_literal: true
 
-step 'Install Bolt package' do
-  bolt_sha = ENV['SHA']
+# Inspired by https://github.com/puppetlabs/pdk/blob/master/package-testing/pre/000_install_package.rb
```

Comment:
@melissa and I plan to chat later about where this could live so both Bolt and PDK can use it.